src/diffusers/__init__.py

```diff
@@ -127,6 +128,7 @@
     )
     _import_structure["schedulers"].extend(
         [
+            "AMusedScheduler",
```

Comment:
```suggestion
            "AMUSEdScheduler",
```
maybe better? not sure @patil-suraj wdyt?